apitools/base/py/credentials_lib.py

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 """Common credentials classes and constructors."""
+from __future__ import unicode_literals
```

Comment:
I'd rather not import this in just one file; I don't think most of the other files in this directory use this import.  (If we're going to use unicode literals, we should do it across the entire library.)